Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -262,36 +262,11 @@ public protocol PurchasesType: AnyObject {
      * If the purchase was successful there will be a ``StoreTransaction`` and a ``CustomerInfo``.
      *
      * If the purchase was not successful, there will be an `NSError`.
-     *
      * If the user cancelled, `userCancelled` will be `true`.
      */
     @objc(purchaseProduct:withCompletion:)
     func purchase(product: StoreProduct, completion: @escaping PurchaseCompletedBlock)
 
-    /**
```

Comment:
As explained in the description, these are moved from `PurchasesType` to `PurchasesSwiftType` because the result type is no longer representable in `Objective-C`. However, we didn't need these `async` APIs in Obj-C as we have the completion block ones.